src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets

```diff
@@ -316,8 +316,7 @@
   <Target Name="ILRepacker"
       BeforeTargets="CopyFilesToOutputDirectory"
       Inputs="$(MSBuildAllProjects);@(IntermediateAssembly);@(InputAssemblies)"
-      Outputs="$(IntermediateOutputPath)ILRepacker.stamp"
-      Condition=" '$(HostOS)' != 'Linux' ">
```

Comment:
Can anyone remember the reason why we disabled this in the first place?

Reply:
It used to crash on Linux under Mono